src/sst/elements/vanadis/vanadis.cc

```diff
@@ -261,6 +262,33 @@ VANADIS_COMPONENT::VANADIS_COMPONENT(SST::ComponentId_t id, SST::Params& params)
     lsq->setRegisterFiles(&register_files);
 
     //////////////////////////////////////////////////////////////////////////////////////
+    rocc0 = loadUserSubComponent<SST::Vanadis::VanadisRoCCInterface>("rocc0");
+
+    if ( nullptr == rocc0 ) {
+        has_rocc0 = false;
+        output->verbose(CALL_INFO, 8, 0, "Error - unable to load a rocc0 accelerator (rocc0 subcomponent)\n");
```

Comment:
It shouldn't produce an error message if you didn't fill the rocc subcomponent slots and didn't intend to. You could put a more targeted error message in if the user attempted to load something and it didn't work out by using [getSubComponentSlotInfo](http://sst-simulator.org/sst-docs/docs/core/component/load/getSubComponentSlotInfo) and not generate an error message if nothing was loaded.